packages/host/app/services/local-realm.ts

```diff
@@ -259,8 +271,8 @@ export default class LocalRealm extends Service {
     return url;
   }
 
-  chooseDirectory(cb?: () => void): void {
-    this.openDirectory.perform(cb);
+  async chooseDirectory(): Promise<void> {
+    await this.openDirectory.perform();
```

Comment:
i wonder if there is even a need anymore to pass a callback to the `openDirectory()` method. if nothing else is passing a callback parameter, then let's just remove the callback parameter from `openDirectory()`

Reply:
Yeah the callback was removed in this PR, is there some other callback that I am missing?